modules/default/compliments/compliments.js

```diff
@@ -1,4 +1,8 @@
 /* global Cron */
+/* global global */
+
+console.log("window name='"+window.name+"'")
```

Comment:
left from debugging?

Reply:
yes for debugging why window.name is empty in test run mode